ansible/group_vars/all

```diff
@@ -105,7 +105,11 @@ limits:
 controller:
   basePort: 10001
   heap: "{{ controller_heap | default('2g') }}"
-  arguments: "{{ controller_arguments | default('') }}"
+  arguments: "{% if metrics.kamon == true %} 
+                {{ controller_arguments + ' -Dkamon.statsd.hostname=' + metrics_kamon_statsd_host + ' -Dkamon.statsd.port=' + metrics_kamon_statsd_port | default('') }} 
+              {% else %} 
+                {{ controller_arguments | default('') }} 
+              {% endif %}"  
```

Comment:
Let's fix these bits, I have an ENV->JVM System property adapter script stashed in another PR. I'll finish that off, then we can clean these up (We need system props for SPIs for example anyway).